public/app/features/logs/components/panel/panel.ts

```diff
@@ -0,0 +1,40 @@
+import { LogListStyle, store } from '@grafana/data';
+
+export function getLogListStyleFromOldProps(
+  wrapLogMessage: boolean | undefined,
+  prettifyJSON: boolean | undefined,
+  logOptionsStorageKey: string | undefined
+) {
+  if (wrapLogMessage === undefined) {
+    wrapLogMessage = store.getBool(`${logOptionsStorageKey}.wrapLogMessage`, false);
+  }
+  if (wrapLogMessage === undefined) {
```

Comment:
should wrapLogMessage be prettifyLogMessage instead? We are doing the same check twice and everything will be set to prettifyJSON.